<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HingeLimits</class>
 <widget class="QWidget" name="HingeLimits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Basis A:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Basis B:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="RotationSetter" name="widgetBasisA" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="RotationSetter" name="widgetBasisB" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Min Limit:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Max Limit:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxMin">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxMax">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioRadius">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Radius</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioDegree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Degree</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RotationSetter</class>
   <extends>QWidget</extends>
   <header>rotationsetter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
